renderer/src/pages/settings/Settings.tsx

```diff
@@ -1,18 +1,30 @@
+import {
+  checkForUpdates,
+  exportSeedPhrase,
+  isOpenAtLogin,
+  saveModuleLogsAs,
+  toggleOpenAtLogin
+} from 'src/lib/station-config'
+
 const Settings = () => {
+  async function getUpdatedCheckedValue (node: HTMLInputElement) {
+    if (node) node.checked = await isOpenAtLogin()
+  }
```

Comment:
Can we please `useState` together with `useEffect` here, instead of manually updating the DOM node? I would like to follow React conventions as closely as possible